src/ol_infrastructure/applications/mitxonline/__main__.py

```diff
@@ -188,9 +188,9 @@
     "MITX_ONLINE_ADMIN_EMAIL": "cuddle-bunnies@mit.edu",
     "MITX_ONLINE_DB_CONN_MAX_AGE": "0",
     "MITX_ONLINE_DB_DISABLE_SSL": "True",  # pgbouncer buildpack uses stunnel to handle encryption"
-    "MITX_ONLINE_FROM_EMAIL": "MITx Online <support@mitxonline.mit.edu>",
+    "MITX_ONLINE_FROM_EMAIL": "MITx Online <mitxonline-support@mit.edu>",
```

Comment:
It looks like this isn't the setting that needs to change. Did some digging locally and there's one called `MAILGUN_FROM_EMAIL` that appears to be the one we use - for some reason I don't see this at all in the config here. (It does exist in Heroku, though.) Changing that has the desired effect. 

On RC this is set to `MITx Online <no-reply@mitxonline-rc-mail.mitxonline.mit.edu>` and on prod this is `MITx Online <no-reply@mail.mitxonline.mit.edu>`. 

(The reply-to setting is fine and is arguably more important.)

Reply:
Should we remove `MITX_ONLINE_FROM_EMAIL` then, and add `MAILGUN_FROM_EMAIL` ?